starknet_py/tests/e2e/block_test.py

```diff
@@ -37,15 +37,12 @@ async def test_block_with_tx_hashes_pending(account):
 # TODO (#1419): Fix contract redeclaration
 @pytest.mark.skip(reason="Redeclaration occurred")
 @pytest.mark.asyncio
-async def test_block_with_tx_hashes_latest(
-    account,
-    map_contract_declare_hash,
-):
+async def test_block_with_tx_hashes_latest(account):
     blk = await account.client.get_block_with_tx_hashes(block_number="latest")
 
     assert isinstance(blk, StarknetBlockWithTxHashes)
     assert isinstance(blk.transactions, list)
-    assert map_contract_declare_hash in blk.transactions
+    assert blk.transactions is not None
```

Comment:
The same reason: https://github.com/software-mansion/starknet.py/pull/1424/files#r1732814169